flaxandteal/coral-arches
# Fix(#2148): undo button breaks id numbers

**Description**

Updates the first or second step of every workflow that has any form of automated ID generation. The undo button has been hidden to prevent it breaking.

**Tests**
- Requires restart, webpack and reload
- Open a workflow through the workflow builder that has had the undo button hidden
- Export the JSON version of the workflow and confirm the undo button being hidden hasn't been lost
- Next go through each commit and check the step/workflow the undo button was hidden on and confirm it is missing on the workflow (I'm not typing them all out)
